helper/userHelper.js

```diff
@@ -0,0 +1,53 @@
+const bcrypt = require("bcrypt");
+const User = require("../models/user");
+const generateToken = require("../utils/generateToken");
+const logger = require("../utils/logger");
+
+const registerHelper = async (req, res) => {
+  try {
+    const { userName, email } = req.body;
+    const foundUser = await User.find({ $or: [{ userName }, { email }] });
+    if (foundUser.length === 0) {
+      const hashPassword = await bcrypt.hash(req.body.password, 10);
```

Comment:
Create a utility function for this.

Reply:
Created utility function userFind to find user